### PR TITLE
Deploy config and scripts update. This prepares for GKE deployment to staging/prod

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: script/deploy_gke.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Script to deploy mixer to a GKE cluster.
+# This retrives the docker images and gRPC descriptor based on git commit hash,
+# so these binaries should have been pushed to container registry and Cloud
+# Storage.
```
Comment (beets) [inline script/deploy_gke.sh:20]: Is there a script that needs to be run to do this? Would be good to reference that here.

Comment (beets) [reply script/deploy_gke.sh:20]: Might be good to also add a warning to do this from a clean repo, and to fetch the latest from upstream (or maybe the script should do that as well?)

Comment (beets) [reply script/deploy_gke.sh:20]: Also check that the ENV is supplied (and exit script if not specified or invalid). The header comment should include possible options.

**Review by beets — APPROVED** [review]

File: script/deploy_gke.sh (hunk 1 above)
Comment (shifucun) [reply script/deploy_gke.sh]: Updated accordingly

**Comment by shifucun** [comment]
thanks for the review!

**Pull request merged**